### ISSUE TITLE
The ID defined in the properties should take precedence over that on the tag

### ISSUE THREAD

**Issue by milelo** [milelo/electric-hiccup#6]
In the https://github.com/weavejester/hiccup implementation,
the element ID defined in the properties takes precedence over that on the tag

**Comment by milelo** [comment]
Fixed in 948675d2f2fb0fb7ecbe137ec99b618e2a868594

**Comment by milelo** [comment]
Released in:
```clojure
        milelo/electric-hiccup {:git/url "https://github.com/milelo/electric-hiccup"
                                :git/tag "v0.2.1-alpha"
                                :git/sha "78fee86"}
```
